include/dxc/Support/HLSLOptions.td

```diff
@@ -265,6 +265,7 @@ def Gfp : Flag<["-", "/"], "Gfp">, HelpText<"Prefer flow control constructs">, F
 // /Gdp - disable effect performance mode - unsupported
 def Ges : Flag<["-", "/"], "Ges">, HelpText<"Enable strict mode">, Flags<[CoreOption]>, Group<hlslcomp_Group>;
 def Gis : Flag<["-", "/"], "Gis">, HelpText<"Force IEEE strictness">, Flags<[CoreOption]>, Group<hlslcomp_Group>;
+def NoLineDirectives : Flag<["-"], "NoLineDirectives">, HelpText<"Not use line directives">, Flags<[CoreOption]>, Group<hlslcomp_Group>;
```

Comment:
I think all the other options support "-" and "/".  Is there a reason to avoid support for "/" here?

Also, you might want to clarify the help text, and possibly even rename the option: "Ignore line directives" and IgnoreLineDirectives.  This makes it clearer that we are ignoring on input, rather than preventing directives on output on a preprocess pass.  This is also consistent with the internal flag name.